concent_api/core/payments/mock.py

```diff
@@ -1,28 +1,13 @@
-from decimal        import Decimal
 from django.conf    import settings
 
 
-def sum_payments(list_of_payments):
-    return sum([item.amount for item in list_of_payments])
-
-
-def get_client_eth_account():
-    return True
-
-
 def get_number_of_eth_block(request = None):  # pylint: disable=inconsistent-return-statements
     if 'HTTP_TEMPORARY_ETH_BLOCK' in request.META:
```

Comment:
I don't see `HTTP_TEMPORARY_ETH_BLOCK` being used anywhere in tests anymore, please remove it.

Reply:
@rwrzesien, I am not aware of it, so I suppose no ;)

Reply:
@dybi They were used to pass customized data to test different cases, so the backend functions don't always return hardcoded values. Please think if this is a functionality you will not need in cluster tests. Or maybe cluster will work on test blockchain using sci_backend, then it shouldn't be needed. But please make a statement - remove it or not.

Reply:
As above I change mock backend to return null values